src/services/enum-utilities.ts

```diff
@@ -251,15 +302,20 @@ export default class EnumUtilities {
       case FilingTypes.CONVERSION: return FilingNames.CONVERSION
       case FilingTypes.CORRECTION: return FilingNames.CORRECTION
       case FilingTypes.COURT_ORDER: return FilingNames.COURT_ORDER
-      case FilingTypes.DISSOLUTION: return FilingNames.DISSOLUTION
+      case FilingTypes.DISSOLUTION:
+        // FUTURE: move dissolution subtype checks here
```

Comment:
I didn't move the dissolution subtype checks here because they check yet another parameter (isFirm), which THIS method doesn't know about. Sigh, the webs we weave.